### ISSUE TITLE
CT error: TypeError: Cannot read property 'firstTime' of undefined

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/friction#96]
friction : accessibility-fuzz : require.js : run
Uncaught TypeError: Cannot read property 'firstTime' of undefined
TypeError: Cannot read property 'firstTime' of undefined
    at Object.alertTemperatureJiggleFromObject (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/friction/js/friction/view/FrictionAlertManager.js?bust=1535068429421:62:24)
    at TemperatureDecreasingDescriber.alertDecrease (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/friction/js/friction/view/describers/TemperatureDecreasingDescriber.js?bust=1535068429421:83:28)
    at Array.TemperatureDecreasingDescriber.amplitudeListener (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/friction/js/friction/view/describers/TemperatureDecreasingDescriber.js?bust=1535068429421:68:16)
    at Emitter.emit2 (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Emitter.js?bust=1535068429421:180:49)
    at NumberProperty._notifyListeners (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Property.js?bust=1535068429421:304:29)
    at NumberProperty.setValueAndNotifyListeners (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Property.js?bust=1535068429421:287:14)
    at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/Property.js?bust=1535068429421:230:16)
    at NumberProperty.set (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/axon/js/NumberProperty.js?bust=1535068429421:86:30)
    at FrictionModel.step (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/friction/js/friction/model/FrictionModel.js?bust=1535068429421:329:30)
    at Sim.stepSimulation (https://bayes.colorado.edu/continuous-testing/snapshot-1535067196651/joist/js/Sim.js?bust=1535068429421:855:22)
Approximately 8/23/2018, 5:33:16 PM

**Comment by jbphet** [comment]
I did some investigating on this, and it appears that the value `alertIndex` is not initially defined in the file `TemperatureDecreasingDescriber`, and this is causing problems within `FrictionAlertManager`.  This code was written and recently added by @zepumph, assigning to him for resolution.

**Comment by zepumph** [comment]
Yes indeed. I'm sorry that you worked on it at all! It has been on my radar but my a11y days are Tuesday and Friday so it had to sit for a bit. Sorry about that!
